docs/source/index.rst

```diff
@@ -77,12 +76,11 @@ The software is available at http://www.aiida.net.
 
 .. toctree::
     :maxdepth: 1
-    :caption: Development
+    :caption: For AiiDA developers
     :hidden:
 
     developer_guide/index
 
-
 .. toctree::
     :maxdepth: 1
     :caption: Tutorials
```

Comment:
Should we also include the sections of `Tutorials` in this restructuring? Moving their content to `StructureData` and `UpfData` respectively? And the `plugin development` sectino should be moved to the temporary `aiida-plugins`, as I think it should be moved out of the `aiida-core` docs

Reply:
I suggest that I mark this in the respective issues, so at least this restructuring is done and we can work on this in the next days during the coding days.
In particular moving tutorials into the section requires probably some thought.

Reply:
For the `plugin development` section, is there something that I have missed, apart for the section that I moved below under the title `aiida-plugins [Section to be moved]`?

Reply:
Added comments for #3019 and #3022 (for the part on `Tutorials`)